packages/web/src/fauceting/utils.ts

```diff
@@ -0,0 +1,59 @@
+import getConfig from 'next/config'
+import { RequestStatus, RequestType } from '../../server/FirebaseClient'
+
+export function getCaptchaKey() {
+  return getConfig().publicRuntimeConfig.RECAPTCHA
+}
+
+export function validateBeneficary(addressOrE164: string, kind: RequestType) {
+  if (kind === RequestType.Invite) {
+    return validateNumber(addressOrE164)
+  } else {
+    return validateAddress(addressOrE164)
+  }
+}
+
+function validateNumber(number: string) {
+  //  TODO use our phone utils from @celo/utils
+  const E164RegEx = /^\+[1-9][0-9]{1,14}$/
+  return E164RegEx.test(number)
+}
+
+export function formatNumber(number: string) {
+  if (number === '+') {
+    return ''
+  }
+  if (number.startsWith('+')) {
+    return number
+  } else {
+    return `+${number}`
+  }
+}
+
+export const EXAMPLE_ADDRESS = 'a0000aaa00a0000...a00a0a0000a00a00aa'
+
+// This is only a basic validation
+function validateAddress(address: string) {
+  return /^(0x)?[0-9a-f]{40}$/i.test(address)
```

Comment:
This should work for hex addresses, can also use `isAddress(address.toLower())` from `web3-utils`. (Lower case it to avoid checksum test.)